### PR TITLE
Adding TimeSpan

### PR DESCRIPTION
**Pull request by rfrerebe**

### CODE REVIEW

File: src/Thoth.Json.Net/Encode.fs [hunk 1]
```diff
@@ -186,6 +186,9 @@ module Encode =
     let datetimeOffset (value : System.DateTimeOffset) : JsonValue =
         JValue(value.ToString("O", CultureInfo.InvariantCulture)) :> JsonValue
 
+    let timespan (value : System.TimeSpan) : JsonValue =
+        JValue(value.ToString()) :> JsonValue
```
Comment (MangelMaxime) [inline src/Thoth.Json.Net/Encode.fs]: Does .Net support `CultureInfo.InvariantCulture` for timespan ? If yes, we need to had it.

Comment (rfrerebe) [reply src/Thoth.Json.Net/Encode.fs]: Alfonso also asked for this. I will check with him.

**Comment by MangelMaxime** [comment]
Released in version 3.0.0-beta-002

**Comment by rfrerebe** [comment]
Hello Maxime,

If you don’t plan to release TimeSpan in v2, I need to revert the change in
doc/v2 about it.

Le me know

Remy

Le ven. 11 janv. 2019 à 11:59, Maxime Mangel <notifications@github.com> a
écrit :

**Comment by MangelMaxime** [comment]
If this is fine to you, I would prefer to release it only in v3.

I already investigated a lot of time to get the whole repo merge and releasing old version require me to jump into the git history and take quite some time and effots...

I will probably explode this repo in an `Thoth` organisation at some point so it's easier to manage package version...

**Pull request merged**